Backendless/Persistence/Polygon.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BackendlessAPI
+{
+  public class Polygon : Geometry
+  {
+    public const String GEOJSON_TYPE = "Polygon";
+    public static String WKT_TYPE = GEOJSON_TYPE.ToUpper();
+
+    private LineString boundary;
+    private List<LineString> holes;
+
+    public Polygon( LineString boundary ) : this( boundary, null, SpatialReferenceSystem.DEFAULT )
+    {
+    }
+
+    public Polygon( List<Point> boundary, List<LineString> holes ) : this( boundary, holes, SpatialReferenceSystem.DEFAULT )
+    {
+    }
+
+    public Polygon( List<Point> boundary, List<LineString> holes, ReferenceSystemEnum srs)
+    : this( new LineString( boundary, srs), holes, srs )
+    {
+    }
+
+    public Polygon( LineString boundary, List<LineString> holes ) 
+    : this( boundary, holes, SpatialReferenceSystem.DEFAULT )
+    {
+    }
+
+    public Polygon( LineString boundary, List<LineString> holes, ReferenceSystemEnum srs )
+    : base( srs )
+    {
+      if ( boundary == null )
+        throw new ArgumentException( "The 'shell' shouldn't be null." );
```

Comment:
shouldn't >> should not
in formal messages it is bad style to shorten words.

Also, what is "The 'shell'" ?? I do not see any shells, users will not see them either.